### ISSUE TITLE
As a visitor, the 'by the numbers' and 'check out these schools' sections have real data

### ISSUE THREAD

**Issue by diego-** [18F/college-choice#280]

**Comment by diego-** [comment]
- [ ] (@diego-) tease out insights from doc for inclusion
- [ ] implement in homepage code

**Comment by LisaGee** [comment]
@diego- I bet you could teach @jjoteal to do this...

**Comment by jjoteal** [comment]
UPDATE: Currently being blocked by Ed/WH on content. Per email subj: "RE: Data Insights/Fun Facts", final content won't be coming until 8/21.

**Comment by jjoteal** [comment]
@ErieMeyer reassigned this to you as I know you are working on selecting the top 3 "by the numbers" data points as well as the top 4-5 "check out these schools" lists. 

Once you have the top 3 "by the numbers" items picked out, please send them my way here so that I can ensure we have relevant illustrations for your top picks.

**Comment by meiqimichelle** [comment]
Done.

**Comment by ErieMeyer** [comment]
This looks great! Opening a new issue for the URLs/search results where this should land.
